src/main/java/activeedge/task/MealTask.java

```diff
@@ -30,7 +30,7 @@ public int getMealCalories() {
 
     @Override
     public String toString() {
```

Comment:
Can you try to fix #227 in the same PR? 

A fix that came to my head is to have an extra function under each task similar to `toString` and use that when displaying in the UI.